### PR TITLE
Os ids

### PR DESCRIPTION
**Pull request by ashryanbeats**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,12 +72,12 @@ For details, see [Registering Your Application](https://creativesdk.adobe.com/do
 
 ## Adding the plugin
 
-Use the command below to add the plugin to your app.
+Use the command below to add the plugin to your app. If you only plan to ship on one platform, you can omit the Client ID and Secret variables for the platform you don't need.
```
Comment (macdonst) [inline README.md]: Unfortunately, you can't omit them but you can put in dummy values.

File: plugin.xml [hunk 2]
```diff
@@ -35,8 +37,8 @@
         <framework src="com.adobe.creativesdk.foundation:auth:0.9.1062+"/>
         <source-file src="src/android/AdobeAuthCredentialsApp.java" target-dir="src/com/adobe/phonegap/csdk" />
         <config-file target="res/values/strings.xml" parent="/resources">
-            <string name="creative_sdk_client_id">$CREATIVE_SDK_CLIENT_ID</string>
-            <string name="creative_sdk_client_secret">$CREATIVE_SDK_CLIENT_SECRET</string>
+            <string name="csdk_client_id_android">$CSDK_CLIENT_ID_ANDROID</string>
```
Comment (macdonst) [inline plugin.xml:40]: This is fine but you don't need to change `name="creative_sdk_client_id"` to `name="csdk_client_id_android"` but that's cool.

File: plugin.xml [hunk 3]
```diff
@@ -52,10 +54,10 @@
         <config-file target="*-Info.plist" parent="CreativeSDK">
             <key>CreativeSDK</key>
             <dict>
-              <key>CSDKClientID</key>
-              <string>$CREATIVE_SDK_CLIENT_ID</string>
-              <key>CSDKClientSecret</key>
-              <string>$CREATIVE_SDK_CLIENT_SECRET</string>
+              <key>CSDKClientID_iOS</key>
```
Comment (macdonst) [inline plugin.xml:57]: Same thing as Android comment.

File: README.md [hunk 4]
```diff
@@ -104,4 +106,6 @@ No action is required for Android. The Creative SDK for Android is delivered as
 <a name="api"></a>
 # Api Guide
 
-There is no JavaScript API for this plugin. All Client Auth code is added to your app automatically.
\ No newline at end of file
+There is no JavaScript API for this plugin. All Client Auth code is added to your app automatically.
+
+If you need to update your Creative SDK Client ID and Secret, or add a new ID and Secret for a new platform, you can do that in this plugin's `plugin.xml` file.
```
Comment (macdonst) [inline README.md]: Nope, you'd change them in the apps `config.xml`. The values get saved like so:

```
<plugin name="phonegap-plugin-csdk-client-auth" spec="https://github.com/CreativeSDK/phonegap-plugin-csdk-client-auth">
        <variable name="CREATIVE_SDK_CLIENT_ID" value="Client ID" />
        <variable name="CREATIVE_SDK_CLIENT_SECRET" value="Client Secret" />
</plugin>
```

File: README.md (hunk 1 above)
Comment (ashryanbeats) [inline README.md]: Thanks! Updating...

File: plugin.xml (hunk 2 above)
Comment (ashryanbeats) [inline plugin.xml:40]: Got it! I just changed these for my human brain. :)

File: README.md (hunk 4 above)
Comment (ashryanbeats) [inline README.md]: Thanks! Updating...

**Comment by macdonst** [comment]
:+1: merge it

**Pull request merged**